palo_alto_cortex_xdr/manifest.json

```diff
@@ -2,15 +2,26 @@
   "manifest_version": "2.0.0",
   "app_uuid": "156afdc8-d8e9-4544-92fd-d8da87278671",
   "app_id": "palo-alto-cortex-xdr",
-  "display_on_public_website": false,
+  "display_on_public_website": true,
```

Comment:
this should be `false` - we are not yet releasing it to the public

Reply:
Done